### PR TITLE
Add / update GH templates for a better user support,

### PR DESCRIPTION
**Pull request by veado**
bug reports, feature requests, issues

- [x] Add template for `user support` 
- [x] Add template for `issues` we use for development 
- [x] Re-order of templates (to have 'user support' on top)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2-Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<!--- Please add steps to reproduce the bug -->
+
+1.
+2.
+3.
+4.
+
+## Possible Solution
+
+<!--- If you have ideas how to fix it, feel free to share these here -->
```
Comment (thatStrangeGuyThorchain) [inline .github/ISSUE_TEMPLATE/2-Bug_report.md]: If you have ideas how to fix it, feel free to share these here
=>
If you have ideas how to fix it, feel free to share them here

File: .github/ISSUE_TEMPLATE/3-Feature_request.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Feature request
+about: You want something added to ASGARDEX.
+labels: 'enhancement'
+---
+
+## Description
+
+<!--- Please describe the enhancement you would like to see in a next version of ASGARDEX -->
```
Comment (thatStrangeGuyThorchain) [inline .github/ISSUE_TEMPLATE/3-Feature_request.md]: Please describe the enhancement you would like to see in a next version of ASGARDEX
=> (not sure)
Please describe the enhancement you would like to see at the next version of ASGARDEX

File: .github/ISSUE_TEMPLATE/4-Issue.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+name: Issue (for development only)
+about: As a developer you want to describe an issue
+---
+
+## Description
+
+<!--- Please describe the issue in few words -->
```
Comment (thatStrangeGuyThorchain) [inline .github/ISSUE_TEMPLATE/4-Issue.md]: Please describe the issue in few words
=> (not sure)
Please describe the issue in a few words

File: .github/ISSUE_TEMPLATE/2-Bug_report.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Your Environment (for using a release version only)
```
Comment (warrenshen) [inline .github/ISSUE_TEMPLATE/2-Bug_report.md]: Your Environment - Release Version (skip if not using a release version)

File: .github/ISSUE_TEMPLATE/2-Bug_report.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+<!--- Please include as many details as possible -->
+
+<!-- required -->
+
+- ASGARDEX desktop version:
+<!--- required -->
+- Operating system and version:
+
+## Your Environment (for development locally only)
```
Comment (warrenshen) [inline .github/ISSUE_TEMPLATE/2-Bug_report.md]: Your Environment - Development Version (skip if not using a development version)

File: .github/ISSUE_TEMPLATE/4-Issue.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+## TODO
+
+<!--- Add todo's to describe all needed steps -->
```
Comment (warrenshen) [inline .github/ISSUE_TEMPLATE/4-Issue.md]: Add TODOs to be done to resolve the issue

**Review by warrenshen — DISMISSED** [review]

**Pull request merged**